<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>844</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>844</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>情报板模拟</string>
  </property>
  <property name="windowIcon">
   <iconset resource="CxResource.qrc">
    <normaloff>:/Ico/MyIco/_Resource/Ico/clap v2.ico</normaloff>:/Ico/MyIco/_Resource/Ico/clap v2.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSplitter" name="splitter_6">
        <property name="minimumSize">
         <size>
          <width>532</width>
          <height>321</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>532</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>480</width>
           <height>129</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>480</width>
           <height>129</height>
          </size>
         </property>
         <property name="title">
          <string>端口</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>串口</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QSplitter" name="splitter_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>端口号</string>
                </property>
               </widget>
               <widget class="QComboBox" name="comboBox"/>
               <widget class="CxLightFrame" name="frame">
                <property name="minimumSize">
                 <size>
                  <width>26</width>
                  <height>26</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>26</width>
                  <height>26</height>
                 </size>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
               </widget>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QSplitter" name="splitter_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <widget class="QPushButton" name="pushButton_3">
                <property name="text">
                 <string>连接</string>
                </property>
               </widget>
               <widget class="QPushButton" name="pushButton_4">
                <property name="text">
                 <string>断开</string>
                </property>
               </widget>
               <widget class="QPushButton" name="pushButton_5">
                <property name="text">
                 <string>端口配置</string>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>网口</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QSplitter" name="splitter">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>IP地址</string>
                </property>
               </widget>
               <widget class="QLineEdit" name="lineEdit">
                <property name="inputMask">
                 <string>000.000.000.000;_</string>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
               <widget class="CxLightFrame" name="frame_3">
                <property name="minimumSize">
                 <size>
                  <width>26</width>
                  <height>26</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>26</width>
                  <height>26</height>
                 </size>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
               </widget>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QSplitter" name="splitter_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <widget class="QPushButton" name="pushButton">
                <property name="text">
                 <string>建立连接</string>
                </property>
               </widget>
               <widget class="QPushButton" name="pushButton_2">
                <property name="text">
                 <string>断开</string>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="minimumSize">
          <size>
           <width>532</width>
           <height>189</height>
          </size>
         </property>
         <property name="title">
          <string>显示区</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QGroupBox" name="groupBox_6">
              <property name="title">
               <string>开关时间</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QSplitter" name="splitter_7">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>开屏时间</string>
                  </property>
                 </widget>
                 <widget class="QTimeEdit" name="timeEdit">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </widget>
               </item>
               <item>
                <widget class="QSplitter" name="splitter_8">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>关屏时间</string>
                  </property>
                 </widget>
                 <widget class="QTimeEdit" name="timeEdit_2">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="CxShowForm" name="shorForm" native="true">
              <property name="minimumSize">
               <size>
                <width>512</width>
                <height>128</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>512</width>
                <height>128</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>下位机时间</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateTimeEdit" name="dateTimeEdit">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_8">
                <property name="text">
                 <string>接上电源</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_7">
                <property name="text">
                 <string>拔电源</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>当前温度</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLCDNumber" name="lcdNumber"/>
              </item>
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>℃</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>消息区</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_6">
           <property name="text">
            <string>清空信息</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>详细信息</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="listWidget">
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>844</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="AccExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="AccSerial"/>
    <addaction name="AccNet"/>
    <addaction name="AccEnvSet"/>
    <addaction name="AccSetledBug"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>管理</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="AccExit">
   <property name="icon">
    <iconset resource="CxResource.qrc">
     <normaloff>:/Ico/MyIco/_Resource/Ico/png-0652.png</normaloff>:/Ico/MyIco/_Resource/Ico/png-0652.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="AccSerial">
   <property name="icon">
    <iconset resource="CxResource.qrc">
     <normaloff>:/Ico/MyIco/_Resource/Ico/png-0089.png</normaloff>:/Ico/MyIco/_Resource/Ico/png-0089.png</iconset>
   </property>
   <property name="text">
    <string>设置串口</string>
   </property>
   <property name="toolTip">
    <string>设置串口</string>
   </property>
  </action>
  <action name="AccNet">
   <property name="icon">
    <iconset resource="CxResource.qrc">
     <normaloff>:/Ico/MyIco/_Resource/Ico/png-0657.png</normaloff>:/Ico/MyIco/_Resource/Ico/png-0657.png</iconset>
   </property>
   <property name="text">
    <string>设置网口</string>
   </property>
  </action>
  <action name="AccEnvSet">
   <property name="icon">
    <iconset resource="CxResource.qrc">
     <normaloff>:/Ico/MyIco/_Resource/Ico/steam.ico</normaloff>:/Ico/MyIco/_Resource/Ico/steam.ico</iconset>
   </property>
   <property name="text">
    <string>环境设置</string>
   </property>
  </action>
  <action name="AccSetledBug">
   <property name="icon">
    <iconset resource="CxResource.qrc">
     <normaloff>:/Ico/MyIco/_Resource/Ico/png-0016.png</normaloff>:/Ico/MyIco/_Resource/Ico/png-0016.png</iconset>
   </property>
   <property name="text">
    <string>设置坏灯泡</string>
   </property>
  </action>
  <action name="action">
   <property name="icon">
    <iconset resource="CxResource.qrc">
     <normaloff>:/Ico/MyIco/_Resource/Ico/png-0048.png</normaloff>:/Ico/MyIco/_Resource/Ico/png-0048.png</iconset>
   </property>
   <property name="text">
    <string>刷新端口</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CxShowForm</class>
   <extends>QWidget</extends>
   <header>cxshowform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CxLightFrame</class>
   <extends>QFrame</extends>
   <header>cxlightframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="CxResource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>listWidget</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>593</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>642</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
